From: Dmitry Shachnev <mitya57@debian.org>
Date: Tue, 22 Oct 2019 23:57:15 +0300
Subject: No longer pass -B to sip

As it is not currently used and won't be in the future
(as there is no plan to allow PyQt5 to be built against Qt6).

Origin: upstream, changeset fd3fc5a1e303
---
 configure.py | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/configure.py b/configure.py
index 7ab57c3..f29271c 100644
--- a/configure.py
+++ b/configure.py
@@ -2560,10 +2560,6 @@ def generate_sip_module_code(target_config, verbose, parts, tracing, mname, fata
     if fatal_warnings:
         argv.append('-f')
 
-    # Make sure any unknown Qt version gets treated as the latest Qt v5.
-    argv.append('-B')
-    argv.append('Qt_6_0_0')
-
     if target_config.prot_is_public:
         argv.append('-P');
 
